nixos/modules/services/cluster/k3s/default.nix

```diff
@@ -35,10 +35,20 @@ in
 
     token = mkOption {
       type = types.str;
```

Comment:
```suggestion
      type = types.nullOr types.str;
```

Reply:
This is too late. It already defaults too an empty string.

Reply:
If fixed it for tokenFile so.